### ISSUE TITLE
Strange Prediction Results

### ISSUE THREAD

**Issue by lrpatterson** [azavea/raster-vision#1113]
So I have this test script where I am training with over 1,000 buildings and a single tif but it is giving me strange prediction results:

**test_preds.png**
![test_preds](https://user-images.githubusercontent.com/41905851/108237169-9f8cf880-7115-11eb-8c64-5eda31ac5fbb.png)

This is my script:
```
import os
from os.path import join
import random


from rastervision.core.rv_pipeline import *
from rastervision.core.backend import *
from rastervision.core.data import *
from rastervision.core.analyzer import *
from rastervision.pytorch_backend import *
from rastervision.pytorch_learner import *
from utils import str_to_bool, save_image_crop




#rastervision run local BFCABTEST.py -a raw_uri "/opt/data/input" -a processed_uri "/opt/data/code/cropped" -a root_uri "/opt/data/output" -a test True

def get_id(p):
    for d,p,files in os.walk(p):
        return [os.path.splitext(f)[0] for f in files ]

def get_config(runner, raw_uri, processed_uri, root_uri, test=False):


    ids = get_id("{}OrthoMosaicTilesLabels".format(raw_uri))

    # val_ids = random.sample(ids,50)
    # train_ids = list(set(ids).difference(set(val_ids)))

    if test:
        # random.shuffle(train_ids)
        # random.shuffle(val_ids)
        #NCCABA012028OrthoSectorTile.
        # train_ids = train_ids[1:2]
        # val_ids = val_ids[1:2]
        #/opt/data/input/OrthoMosaicTiffLimited/Rasters_NCCABA018018OrthoSectorTile.tif
        train_ids = ['Rasters_NCCABA018019OrthoSectorTile']
        val_ids = ['Rasters_NCCABA018026OrthoSectorTile']

    class_config = ClassConfig(
        names=[
            'Building', "Background"
        ],
        colors=[
            '#ffff00', '#ffffff'
        ])

    def make_scene(id):
        raster_uri = '{}/OrthoMosaicTiffLimited/{}.tif'.format(raw_uri, id)

        label_uri = '{}/OrthoMosaicTilesLabels/{}.geojson'.format(raw_uri, id.split('_')[1])
        print(raster_uri, label_uri, sep=" ")


        channel_order = [0,1,2]
        raster_source = RasterioSourceConfig(
            uris=[raster_uri], channel_order=channel_order)

        vector_source = GeoJSONVectorSourceConfig(
            default_class_id=0, uri=label_uri, ignore_crs_field=True)
        label_source = SemanticSegmentationLabelSourceConfig(
            raster_source=RasterizedSourceConfig(
                vector_source=vector_source,
                rasterizer_config=RasterizerConfig(background_class_id=1)))

        scene = SceneConfig(
            id=id,
            raster_source=raster_source,
            label_source=label_source
            )

        return scene

    dataset = DatasetConfig(
        class_config=class_config,
        train_scenes=[make_scene(id) for id in train_ids],
        validation_scenes=[make_scene(id) for id in val_ids])
    chip_sz = 300
    chip_options = SemanticSegmentationChipOptions(
        window_method=SemanticSegmentationWindowMethod.sliding, stride=chip_sz)

    backend = PyTorchSemanticSegmentationConfig(
        model=SemanticSegmentationModelConfig(backbone=Backbone.resnet50),
        solver=SolverConfig(
            lr=1e-4,
            num_epochs=10,
            test_num_epochs=2,
            batch_sz=8,
            one_cycle=True),
        log_tensorboard=True,
        run_tensorboard=True,
        test_mode=test)

    return SemanticSegmentationConfig(
        root_uri=root_uri,
        dataset=dataset,
        backend=backend,
        train_chip_sz=chip_sz,
        predict_chip_sz=chip_sz,
        chip_options=chip_options)

```
What am I doing wrong on this?

Here is a sample of my data: https://cabarruscountync-my.sharepoint.com/:f:/g/personal/lrpatterson_cabarruscounty_us/EkKdlxDGB5BEiWbN4KH5Z6ABOogv3RGS8EBhpYg9480FSA?e=WF4bSC

**Comment by AdeelH** [comment]
What version of raster vision are you using? 

Are the results strange because you know that the model is good? Does that mean that the training completes successfully i.e. the validation loss goes down and the metrics improve during training?

Do the images and labels look correct and aligned in the samples in `/opt/data/output/dataloaders`?

**Comment by AdeelH** [comment]
Closing this for now. Please reopen if there are any more updates or diagnostics or questions.
